homepage/forms.py

```diff
@@ -35,17 +35,26 @@ class FilterForm(forms.Form):
 
 # This class uses the built in django UserCreationForm and adds an email field
 class SignUpForm(UserCreationForm):
-    email = forms.EmailField(required=True)
+    email = forms.EmailField(initial='@mta.ac.il', help_text='Required. must use an academic email.', required=True)

```

Comment:
need to think about other tests which use invalid mail as valid.
maybe the part of them in the list below:
homepage/migrations/0002_user_test_data.py:15:            ('testUser1', 'user1@gmail.com', 'password123'),
homepage/migrations/0002_user_test_data.py:16:            ('testUser2', 'David@gmail.com', 'Password456'),
homepage/migrations/0002_user_test_data.py:17:            ('testUser3', 'User3@gmail.com', 'User3Password')
homepage/tests/sign_in_tests.py:13:        User.objects.create_user(username='valid_username', email='valid@email.com', password='pw123123')
homepage/tests/sign_out_tests.py:10:        User.objects.create_user(username='valid_username', email='valid@email.com', password='pw123123')
homepage/tests/followedUserCourses_model_tests.py:13:        app_user1 = AppUser.create_app_user('user1', 'user2@gmail.com', '1234')

please check it.

Reply:
Fixed all occurrences